arches_component_lab/src/arches_component_lab/widgets/DatePickerWidget/components/DatePickerWidgetEditor.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+import { ref, watchEffect } from "vue";
+
+import { convertISO8601DatetimeFormatToPrimevueDatetimeFormat } from "@/arches_component_lab/widgets/DatePickerWidget/utils.ts";
+
+import DatePicker from "primevue/datepicker";
+import GenericFormField from "@/arches_component_lab/generic/GenericFormField.vue";
+
+import { formatDate } from "@/arches_component_lab/datatypes/date/utils.ts";
+
+import type { FormFieldResolverOptions } from "@primevue/forms";
+import type { DateDatatypeCardXNodeXWidgetData } from "@/arches_component_lab/datatypes/date/types.ts";
+import type { NodeData } from "@/arches_component_lab/types.ts";
+
+const { value, nodeAlias, cardXNodeXWidgetData } = defineProps<{
+    value: NodeData | null | undefined;
+    nodeAlias: string;
+    cardXNodeXWidgetData: DateDatatypeCardXNodeXWidgetData;
+}>();
+
+const shouldShowTime = ref(false);
+const dateFormat = ref();
+
+watchEffect(() => {
+    const convertedDateFormat =
+        convertISO8601DatetimeFormatToPrimevueDatetimeFormat(
+            cardXNodeXWidgetData.config.dateFormat,
```

Comment:
if `dateFormat` doesn't exist on the cardXNodeXWidgetData.config we need to pick it up from the cardXNodeXWidgetData.node.config
```suggestion
            cardXNodeXWidgetData.config.dateFormat || cardXNodeXWidgetData.node.config.dateFormat,
```

Reply:
ah that's cruft from a previous implementation. Removed :D